dotcom-rendering/src/components/LiveblogRightAdSlots.importable.tsx

```diff
@@ -0,0 +1,36 @@
+import { useAB } from '../lib/useAB';
+import { AdSlot } from './AdSlot';
+import { LiveblogRightMultipleAdSlots } from './LiveblogRightMultipleAdSlots';
+
+type Props = {
+	display?: ArticleDisplay;
+	isPaidContent?: boolean;
+};
+
+export const LiveblogRightAdSlots = ({ display, isPaidContent }: Props) => {
+	const ABTestAPI = useAB()?.api;
+	const shouldInsertMultipleAdverts =
+		ABTestAPI?.isUserInVariant(
+			'LiveblogRightColumnAds',
+			'multiple-adverts',
+		) ?? false;
+
+	if (shouldInsertMultipleAdverts) {
+		return (
+			<LiveblogRightMultipleAdSlots
+				display={display}
+				isPaidContent={isPaidContent}
+			/>
+		);
+	}
+
+	return (
+		<AdSlot
+			data-right-ad="1"
```

Comment:
What's this `data-right-ad` attribute for? If it's required for something, could you add a comment to explain why?

Reply:
Never seen that before in my life... (I must have been using it for testing)

Good spot!